src/libsyntax_ext/proc_macro_harness.rs

```diff
@@ -404,6 +404,11 @@ fn mk_decls(
     ).map(|mut i| {
         let attr = cx.meta_word(span, sym::rustc_proc_macro_decls);
         i.attrs.push(cx.attribute(attr));
+
+        let deprecated_nested_attr = cx.meta_nested_word(span, sym::deprecated);
+        let allow_deprecated_attr = cx.meta_list(span, sym::allow, vec![deprecated_nested_attr]);
+        i.attrs.push(cx.attribute(allow_deprecated_attr));
```

Comment:
Did you mean to push these changes to this PR?

Reply:
Sorry for the wrong commit. I have just reverted it and have created another pr #65484. Could you please r+ again for me?